content/blog/doc-event-driven-api.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "API documentation in event-driven applications"
+date: 2020-05-12T10:00:10+01:00
+type: blog
+tags:
+  - AsyncAPI
+  - Event-Driven Architectures
+  - EDA
+cover: /images/posts/doc-event-driven-api.webp
+weight: 200
```

Comment:
please add metadata `featured: true` so it will show up on the top of the blog landing page